static/src/javascripts/projects/commercial/modules/lotame-data-extract.js

```diff
@@ -17,14 +20,10 @@ const shouldLoadLotame = (): boolean => {
         (edition === 'UK' || edition === 'INT')
     );
 };
+let initialised: boolean = false;
 
 // Fetches Lotame Data for the Ozone project
-// and stores in in window.OzoneLotameData
-const init = (): Promise<void> => {
-    if (!shouldLoadLotame || commercialFeatures.isSecureContact) {
-        return Promise.resolve();
-    }
-    return loadScript('//ad.crwdcntrl.net/5/c=13271/pe=y/var=OzoneLotameData')
+const loadLotamePromise = () => loadScript('//ad.crwdcntrl.net/5/c=13271/pe=y/var=OzoneLotameData')
         .then(() => {
             if (config.get('isDotcomRendering', false)) {
                 // We do not need the LOTCC initialization for dotcom-rendering Ad Free
```

Comment:
Why is there an assumption that this would only be Ad free?

Reply:
All that was existing logic, i just extracted it into a function to be used in TCFv2 too. 🤷🏻‍♀️

Reply:
Fair enough. I was just curious what this logic implied…